server/commands/deployments/DeployService.js

```diff
@@ -44,11 +54,51 @@ module.exports = function DeployServiceCommandHandler(command) {
 
 function validateCommandAndCreateDeployment(command) {
   return co(function* () {
+    const { mode, packagePath, environmentName, serviceSlice, serviceName, serviceVersion } = command;
```

Comment:
that's nice. You could also put that as a function argument if you prefer, and remove "command" var